save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/OrganizationRepository.kt

```diff
@@ -35,9 +35,22 @@ ValidateRepository {
      */
     fun findByNameStartingWithAndStatus(prefix: String, status: OrganizationStatus): List<Organization>
 
+    /**
+     * @param prefix prefix of organization name
+     * @param status
+     * @return list of organizations with names that start with [prefix]
+     */
+    fun findByNameStartingWithAndStatusIn(prefix: String, status: List<OrganizationStatus>): List<Organization>
+
     /**
      * @param status
      * @return list of organizations with required status
      */
     fun findByStatus(status: OrganizationStatus): List<Organization>
+
+    /**
+     * @param status
```

Comment:
Please rename to `statuses` and document.

Reply:
Done